crates/turborepo-lib/src/run/task_id.rs

```diff
@@ -8,7 +8,7 @@ pub const TASK_DELIMITER: &str = "#";
 pub const ROOT_PKG_NAME: &str = "//";
 
 /// A task identifier as it will appear in the task graph
-#[derive(Debug, Clone, Hash, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Debug, Clone, Hash, PartialEq, Eq, PartialOrd, Ord, Serialize)]
```

Comment:
I think that we would want to serialize by first going through `String`. I'm not sure where this is getting used in this PR, but I would expect we want `"web#build"` and not `{ "package": "web", "task": "build" }`